src/Skia/Avalonia.Skia/SkiaSharpExtensions.cs

```diff
@@ -95,6 +100,11 @@ public static Rect ToAvaloniaRect(this SKRect r)
         {
             return new Rect(r.Left, r.Top, r.Right - r.Left, r.Bottom - r.Top);
         }
+        
+        public static PixelRect ToAvaloniaPixelRect(this SKRectI r)
```

Comment:
Note: all of these extensions are public for some reason. But it's too late to change that.